nightlies/cron_wrapper

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# /nightlies/cron_wrapper.sh
+
+if [ -z $1 ]; then
+  echo "need argument"
+  exit 1
+fi
+
+BASENAME=`basename $0`
+LOG="/var/tmp/$BASENAME.$$.txt"
+
+"$@" &>"$LOG"
+
+rc=$?
+
+#if there is an error, eamil to MAILTO in crontab
+if [ $rc != 0 ]; then
+        echo "return code $rc"
+        cat "$LOG"
+fi
+
+#trap "rm -f $TMP" 0
+rm -f $LOG
```

Comment:
My intent was on errors only, unless someone suggests otherwise.  We could store success logs somewhere, but then we will need not to forget to clean them up